roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml

```diff
@@ -67,7 +67,7 @@ platforms:
     groups:
       - kafka_rest
     image: geerlingguy/docker-centos7-ansible
-    dockerfile: ../Dockerfile.j2
+    # dockerfile: ../Dockerfile.j2
```

Comment:
Please remove this line completely, it's not necessary if we don't use the docker file.